src/components/Layout/PhoneNumberInput.tsx

```diff
@@ -77,8 +88,13 @@ export const PhoneNumberInput: FunctionComponent<{
         <TextInput
           style={styles.numberInput}
           keyboardType="phone-pad"
-          value={mobileNumberValue}
-          onChange={({ nativeEvent: { text } }) => onChangeMobileNumber(text)}
+          value={value}
+          onChange={({ nativeEvent: { text } }) => {
```

Comment:
Noticed `onChange` being used but there's `onChangeText`, perhaps that is a better callback to use?
Let me know what you think!

Reply:
`onChange` to `onChangeText` Because `onChangeText` use text then no need to destructure object compared to `onChange`

Reply:
the "SG" also reformatted.

Reply:
The change to `onChangeText` looks good!